cypress/support/step_definition/login_management/forgotPasswordDialogSteps.spec.js

```diff
@@ -0,0 +1,29 @@
+import Login_Management from '../../pages/login_management/pageLoginManagement'
+
+const loginManagement = new Login_Management()
+
+//Scenario: User sees password recovery dialog
+//Given I am on the dBildungscloud login page as a user on BRB
+//this step defined in --> \step_definition\login_management\commonLoginManagementRelatedSteps.spec.js
+
+//When I click on Forgot Password? link
+//this step defined in --> \step_definition\login_management\commonLoginManagementRelatedSteps.spec.js
+
+Then('I see the Reset Password dialog with username label, email input box, info message, cancel and send reset password buttons', () => {
+  loginManagement.showElementOnDialog()
+})
+
+//Scenario: Entering email or user name is mandatory
+// Given I am on the dBildungscloud login page as a user on BRB
+//this step defined in --> \step_definition\login_management\commonLoginManagementRelatedSteps.spec.js
+
+//When I click on Forgot Password? link
+//this step defined in  --> \step_definition\login_management\commonLoginManagementRelatedSteps.spec.js
+
+When('I remove my email from the input box and submit the request', () => {
```

Comment:
`When('I clear email section and submit the request' `

Reply:
updated!